purchase_requisition_from_rfq/models/purchase_requisition.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class PurchaseRequisition(models.Model):
+    _inherit = "purchase.requisition"
+
+    purchase_ids = fields.One2many(
+        default=lambda self: self._context.get('active_ids', []),
+    )
+    line_ids = fields.One2many(
+        default=lambda self: self._get_line(),
+    )
+
+    @api.multi
+    def _get_line(self):
+        order = orders = self.env['purchase.order'].browse(
+            self._context.get('active_ids', []))
+        if orders.mapped('requisition_id'):
+            raise ValidationError(_(
+                'Order: %s, already agreement created!') % ', '.join(
+                orders.filtered(lambda l: l.requisition_id).mapped('name')))
+        dic = {}
+        len_max = 0
+        for order in orders:
+            if order.state != 'draft':
+                raise ValidationError(_('The purchase agreement cannot be '
+                                        'processed because the order is '
+                                        'not draft!'))
+            dic[order.id] = [(line.product_id.id, line.product_qty)
```

Comment:
Just add comment, to tell what we try to do.
```suggestion
            # Test combination of product and quantity
            dic[order.id] = [(line.product_id.id, line.product_qty)
```